wdl-engine/src/config.rs

```diff
@@ -448,6 +455,136 @@ impl Default for DockerBackendConfig {
     }
 }
 
+/// Represents information for TES backend basic authentication.
+#[derive(Debug, Default, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "snake_case", deny_unknown_fields)]
+pub struct TesBackendBasicAuth {
```

Comment:
Nit: does this really need to be qualified with `TesBackend`? Seems that struct for holding basic auth would be generally useful for other servers later on.